kaska/tests/test_smoothn.py

```diff
@@ -13,76 +13,86 @@
 
```

Comment:
Please revert this, as it is heavily changed as part of my smoothn changes, and I don't wnat to have to fix the conflicts. There are PEP8 changes I will have to make, based on this, though.